cns/cnsclient/cli.go

```diff
@@ -96,3 +101,20 @@ func printIPAddresses(addrSlice []cns.IPConfigurationStatus) {
 		cns.IPConfigurationStatus.String(addr)
 	}
 }
+
+func getInMemory(client *CNSClient) error {
+
+	inmemoryData, err := client.GetHTTPServiceData()
+	if err != nil {
+		return err
+	}
+
+	printInMemoryStruct(inmemoryData.HttpRestServiceData)
+	return nil
+}
+
+func printInMemoryStruct(data restserver.HttpRestServiceData){
+	 fmt.Println("PodIPIDByOrchestratorContext : ", data.PodIPIDByOrchestratorContext)
+	 fmt.Println("PodIPConfigState : ", data.PodIPConfigState)
+	 fmt.Println("IPAMPoolMonitor : ", data.IPAMPoolMonitor)
```

Comment:
nit: no space before colon

Reply:
I thought giving the space would make the CLI o/p more readable, but makes sense